docker-compose.yml

```
@@ -5,19 +5,19 @@ services:
       context: ./php-apache
       dockerfile: Dockerfile
     ports:
-      - [ 1 ]:[ 2 ]
+      - 8080:80
     volumes:
       - type: bind
-        source: [ 3 ]
+        source: "./php-apache/000-default.conf"
         target: "/etc/apache2/sites-enabled/000-default.conf"
       - type: bind
         source: "./php-apache/app"
-        target: [ 4 ]
+        target: "/var/www/docker_review"
     depends_on:
       - review-mysql
 
   review-mysql:
-    image: [ 5 ]
+    image: mysql:5.7
```

訂正は不要ですが、
問題の方に「MySQLコンテナのイメージは 8 を使用してください。」とあるので正しくは下記ですね 👀 

Mysqlのバージョンが違うと使用できるコマンドが違ったりしてエラーの原因になりやすいので、どのバージョンを使用するかは大事です。

```suggestion
    image: mysql:8
```